### DISCUSSION TITLE
How to change formatting with 'Beatify'  for wrapping

### DISCUSSION THREAD

**Discussion by chartnet** [phcode-dev/phoenix#1947] [Q&A]
Hello,

I m trying to use Phoenix Code with the built-in beautify function. The function is wrapping certain text which is anoying to read. For example CSS: https://prnt.sc/CAVm-RmFKCw3

These should be one-liners.

Is there a way to change this?

**Comment by abose** [comment]
https://docs.phcode.dev/docs/Features/beautify-code#customizing-beautify-code

These are the available options for beautify right now. We use prettier under the hood, so anything prettier supports can be done.
